tests/upgrade/0.8.2-0.9.0/jiva/volume_test.go

```diff
@@ -72,6 +78,39 @@ var _ = Describe("[jiva] TEST VOLUME UPGRADE", func() {
 			podCount := ops.GetPodRunningCountEventually("default", upgradedLabel, replicaCount+1)
 			Expect(podCount).To(Equal(replicaCount+1), "while checking pod version")
 
+			By("verifying registered replica count and replication factor")
+			podList, err := ops.PodClient.List(metav1.ListOptions{LabelSelector: ctrlLabel})
+			Expect(err).ShouldNot(HaveOccurred(), "while listing controller pod")
+
+			replicationFactor := ""
+			for _, env := range podList.Items[0].Spec.Containers[0].Env {
```

Comment:
Let us search container by name

Reply:
We might think to create builder code at `pkg/kubernetes/container/v1alpha1`
Probably in next PRs

Reply:
Yes in the next PR will try to have builder for container